addons/controllers/antreaconfig_controller_test.go

```diff
@@ -227,4 +254,212 @@ var _ = Describe("AntreaConfig Reconciler and Webhooks", func() {
 			Expect(k8sClient.Update(ctx, config)).ToNot(Succeed())
 		})
 	})
+
+	When("antrea config is precreated with {CLUSTER_NAME}-{package-short-name}-package", func() {
```

Comment:
"antrea config is precreated with name {CLUSTER_NAME}-{package-short-name}-package